web/src/hashchange.js

```diff
@@ -323,7 +325,9 @@ function do_hashchange_overlay(old_hash) {
     }
 
     if (base === "streams") {
-        stream_settings_ui.launch(section);
+        // e.g. #streams/29/social/subscribers
+        const right_section = hash_parser.get_current_nth_hash_section(4);
+        stream_settings_ui.launch(section, right_section);
```

Comment:
`right_side_tab` might be a better name than `right_section` for the selected tab.

Reply:
It was named alongside the `left_section` variable, would you like to rename that to `left_side_tab` as well? I'm going to leave it as is for now since I think it's better to have `left_section` and `right_section` than `left_section` and `right_side_tab` (makes the left variable seem like it's more than just the tab)

Reply:
Yes, that's probably better. Though I will note that the `left_section/left_side_tab` variable that this PR introduces is somewhat incoherent -- the previous singleton `section` variable could be "new", which is OK, but "new" is a state of the right side, not of the left side (and I think we just preserve which tab on the left side is open?).

I think we want to pass `"subscribed", "new"` after parsing `new` from the URL to correct that, since it looks like `"new"` is always shown with the "subscribed" tab on the left panel.

Reply:
Oh, I forgot I'd renamed `left_section` 🤦 sorry about that. I can rename them both to `X_side_tab`.

Reply:
`"new"` isn't shown with "subscribed" in the left panel if the popover is already open and in the "all streams" section

It might actually make sense to keep it as `section` and have the right side tab sometimes null? Because the `section` can also sometimes be a stream identifier.

Reply:
That's probably workable, but we should open a follow-up issue; fundamentally, the `new` state is about the right panel, not the left one.

Reply:
Opened an issue #27730